### PR TITLE
Prepare Redux to support Ledger accounts

### PR DESCRIPTION
**Pull request by cairomassimo**
Refactoring-only PR.

Prepares Redux code for signing with methods other than keyrings.

### CODE REVIEW

File: background/redux-slices/selectors/accountsSelectors.ts [hunk 1]
```diff
@@ -195,35 +196,45 @@ export type AccountTotal = {
...
+}
+
 export const selectAccountTotalsByCategory = createSelector(
   getAccountState,
   getAssetsState,
-  selectSigningAddresses,
-  (accounts, assets, signingAddresses): CategorizedAccountTotals => {
+  selectAddressSigningMethods,
+  (accounts, assets, signingAccounts): CategorizedAccountTotals => {
+    // TODO: here
```
Comment (greg-nagy) [inline background/redux-slices/selectors/accountsSelectors.ts:220]: This will be important in the future! :)

**Review by greg-nagy — APPROVED** [review]
This looks good 🚀

File: background/redux-slices/selectors/accountsSelectors.ts [hunk 2]
```diff
@@ -195,35 +196,45 @@ export type AccountTotal = {
...
   accountType: AccountType
+  signingMethod: SigningMethod | null
   name?: string
   avatarURL?: string
   localizedTotalMainCurrencyAmount?: string
 }
 
 export type CategorizedAccountTotals = { [key in AccountType]?: AccountTotal[] }
 
+const signingMethodTypeToAccountType: Record<
```
Comment (greg-nagy) [inline background/redux-slices/selectors/accountsSelectors.ts:207]: I really like this typing 💛

File: background/redux-slices/selectors/accountsSelectors.ts (hunk 1 above)
Comment (cairomassimo) [reply background/redux-slices/selectors/accountsSelectors.ts:220]: Whoops, it was a comment to myself. Forgot to remove it!

**Pull request merged**